src/components/Pagination.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import S from './Pagination.module.css';
+
+export function Pagination({ pageLimit, defaultPage = 1, onChange }) {
+  const [current, setCurrent] = useState(defaultPage);
+
+  useEffect(() => {
+    onChange && onChange(current);
+  }, [current]);
+
+  const handelNextPage = () => {
+    if (current > 1) {
+      setCurrent(next => next + 1);
+    }
+  };
+
+  const handlePrePage = () => {
+    if (current < pageLimit) {
+      setCurrent(prev => prev - 1);
+    }
+  };
```

Comment:
```suggestion
  const handlePrevPage = () => {
    if (current < pageLimit) {
      setCurrent(current => current - 1);
    }
  };
```
마찬가지 입니다!

또한 1보다 작을 때는 막아줘야할 것 같아요!